packages/kbn-type-summarizer/jest.integration.config.js

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+/** @typedef {import("@jest/types").Config.InitialOptions} JestConfig */
+/** @type {JestConfig} */
+module.exports = {
+  preset: '@kbn/test/jest_integration_node',
+  rootDir: '../..',
+  roots: ['<rootDir>/packages/kbn-type-summarizer'],
+  watchPathIgnorePatterns: ['<rootDir>/packages/kbn-type-summarizer/tests/__tmp__/.*'],
```

Comment:
Should we ignore `__tmp__` as part of the [preset](https://github.com/elastic/kibana/blob/main/packages/kbn-test/jest-preset.js)?

Reply:
Sounds like a reasonable idea